tests/phpunit/tests/media.php

```diff
@@ -4339,6 +4330,175 @@ public function test_wp_get_loading_optimization_attributes( $context ) {
 		}
 	}
 
+	/**
+	 * Tests that wp_get_loading_optimization_attributes() returns fetchpriority=high and increases the count for arbitrary contexts in the main loop.
+	 *
+	 * @ticket 58894
+	 *
+	 * @covers ::wp_get_loading_optimization_attributes
+	 *
+	 * @dataProvider data_wp_get_loading_optimization_attributes_arbitrary_contexts
+	 *
+	 * @param string $context
```

Comment:
```suggestion
	 * @param string $context Context for the element for which the loading optimization attribute is requested.
```
Let's add a description for this parameter.